DQMOffline/Trigger/src/HigPhotonJetHLTOfflineSource.cc

```diff
@@ -322,7 +322,8 @@ HigPhotonJetHLTOfflineSource::endRun(const edm::Run & iRun,
 {
   // Normalize to the total number of events in the run
   TH2F* h = trigvsnvtx_->getTH2F();
-  double norm = evtsrun_*hltPathsToCheck_.size()/h->Integral(); 
+  double integral = h->Integral();
+  double norm = (integral > 0.) ? evtsrun_*hltPathsToCheck_.size()/integral : 1.;
```

Comment:
sorry, I'm confused
shouldn't such a computation be done in the HARVESTING step ?